src/OpenLoco/src/Drawing/SoftwareDrawingContext.cpp

```diff
@@ -54,6 +54,7 @@ namespace OpenLoco::Drawing
         static PaletteMap::Buffer<8> _textColours{ 0 };
         static uint16_t getStringWidth(const char* buffer);
         static std::pair<uint16_t, uint16_t> wrapString(char* buffer, uint16_t stringWidth);
+        static uint16_t wrapStringTicker(char* buffer, uint16_t stringWidth, uint16_t numCharacters);
```

Comment:
Could we perhaps unify wrapString and wrapStringTicker? What are the fundamental differences between the two?

Reply:
it has different returns and different input. It could be doable but its not going to be easy. You would likely have to do wrap string then have another function that walks the wrap string to work out where the num characters line would end up.